linux_os/guide/system/software/integrity/crypto/configure_openssl_tls_crypto_policy/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Configure OpenSSL library to use TLS Encryption'
+
+description: |-
+    Crypto Policies are means of enforcing certain cryptographic settings for
+    selected applications including OpenSSL. OpenSSL is by default configured to
+    modify its configuration based on currently configured Crypto Policy.
+    However, in certain cases it might be needed to override the Crypto Policy
+    specific to OpenSSL and leave rest of the Crypto Policy intact.
+
+rationale: |-
+    Without cryptographic integrity protections, information can be altered by
+    unauthorized users without detection.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: CCE-84255-9
+
+references:
+    nist: AC-17(2)
```

Comment:
You're right.. the update-crypto-policies probably shouldn't run - this is something that should only be ran when setting a new crypto-policy, otherwise it will place all the default `/etc/crypto-policies/back-ends/...config` to their default state... I'll clean up to use the lineinfile template.